### DISCUSSION TITLE
Uporządkowanie numerów zwracanych przez phfwdReverse

### DISCUSSION THREAD

**Discussion by Alef-Zero** [Kwasow/IPP_testy#29] [General]
Wynikowe numery mają być posortowane leksykograficznie.
Dopóki nie było cyfr 10(*) i 11(#) było to jednoznaczne.
Pytanie czy numer ***1** jest przed numerem **91** czy za nim? Jeżeli porządkować według kodów ASCII to ***1** jest przed **91** ale jeżeli rozpatrywać wartości kolejnych cyfr to **91** jest przez ***1**.

**Comment by Kwasow** [comment]
Wydaje mi się, że to jest dobre pytanie na forum, bo dla mnie bardziej naturalnie leksykograficznie to jest *9 po 19, ale argument z ASCII też jest dobry.
